<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2B2B2B;
    color: #FFFFFF;
    border: none;
}

QPushButton {
    background-color: #3A3A3A;
    border: 1px solid #5C5C5C;
    border-radius: 5px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #505050;
}

QPushButton:pressed {
    background-color: #3A3A3A;
    border: 1px solid #76c7c0;
}

QSlider::groove:horizontal {
    border: 1px solid #5C5C5C;
    height: 8px;
    background: #3A3A3A;
}

QSlider::handle:horizontal {
    background: #76c7c0;
    border: 1px solid #5C5C5C;
    width: 14px;
    margin: -3px 0;
    border-radius: 7px;
}
QPushButton {
    background-color: #4CAF50; /* 亮绿色 */
    border: 1px solid #76c7c0;
    border-radius: 5px;
    padding: 5px;
    font-weight: bold; /* 加粗字体 */
    color: white; /* 字体颜色设为白色 */
}

QPushButton:hover {
    background-color: #45a049; /* 鼠标悬停时的颜色 */
}

QPushButton:pressed {
    background-color: #367c39; /* 按下按钮时的颜色 */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="videoWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QSlider" name="videoSlider">
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: #333333;
    margin: 0px 0;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: #4CAF50;
    border: 1px solid #4CAF50;
    width: 14px;
    margin: -3px 0;
    border-radius: 7px;
}

QSlider::sub-page:horizontal {
    background: #76c7c0; /* 已播放部分的颜色 */
    border: 1px solid #76c7c0;
    height: 8px;
    border-radius: 4px;
}

QSlider::add-page:horizontal {
    background: #333333; /* 未播放部分的颜色 */
    border: 1px solid #333333;
    height: 8px;
    border-radius: 4px;
}
</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="speedComboBox">
            <property name="styleSheet">
             <string notr="true">QComboBox {
    background-color: #4CAF50; /* 主体背景颜色 */
    border: 1px solid #76c7c0;
    border-radius: 5px;
    padding: 5px;
    font-weight: bold;
    color: white;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #76c7c0;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
    background-color: #367c39;
}

QComboBox::down-arrow {
    image: url(:/icons/arrow_down.png); /* 使用自定义图标 */
    width: 14px;
    height: 14px;
}

QComboBox QAbstractItemView {
    background-color: #333333; /* 下拉列表背景颜色 */
    border: 1px solid #76c7c0;
    selection-background-color: #45a049; /* 选中项的背景颜色 */
    selection-color: white; /* 选中项的字体颜色 */
    color: white;
}

QComboBox QAbstractItemView::item {
    height: 30px; /* 每一项的高度 */
    padding: 0 5px;
}
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="addVideoButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>加载视频</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeVideoButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>删除视频</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveUpButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>上移视频</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveDownButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>下移视频</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="videoListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
